spec/02-integration/14-tracing/02-propagation_spec.lua

```diff
@@ -160,7 +160,7 @@ for _, strategy in helpers.each_strategy() do
         assert.is_same(0, #spans, res)
 
         local traceparent = assert(body.headers.traceparent)
-        assert.equals("00-" .. trace_id .. "-" .. span_id .. "-00", traceparent)
+        assert.matches("00%-" .. trace_id .. "%-%x+%-00", traceparent)
```

Comment:
This is a side effect of the created span that is no longer noop, so it is now used for propagation along with the `00` sampled flag.